roles/local_facts/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+- name: create directory for ansible custom facts
```

Comment:
Why is this it's own role instead of a task in the minion role?  It seems that this only applies to minions, so it seems like it would make more sense in the minion role than out on it's own. Also 'local_facts' is way too generic of a name if this only applies to minions.